app/src/main/java/com/artemzin/qualitymatters/models/PicassoBitmapClient.java

```diff
@@ -0,0 +1,21 @@
+package com.artemzin.qualitymatters.models;
+
+import android.widget.ImageView;
+import com.squareup.picasso.Picasso;
+
+import javax.inject.Inject;
+
+public class PicassoBitmapClient implements NetworkBitmapClient {
+
+  private final Picasso picasso;
+
+  @Inject
```

Comment:
ummm, @artem-zinnatullin I think `@Inject` here just the way we shorten our code so that we can provide it like this:
````
@Provides @NonNull @Singleton
    public QualityMattersImageLoader provideNetworkBitmapClient(@NonNull PicassoImageLoader picassoBitmapClient) {
        return picassoBitmapClient;
    }
````
instead of like this:
````
@Provides @NonNull @Singleton
    public QualityMattersImageLoader provideNetworkBitmapClient(@NonNull Picasso picasso) {
        return new PicassoImageLoader(picasso);
    }
````